gateway-service-api/build.gradle.kts

```diff
@@ -15,14 +15,14 @@ val generateLocalGoGrpcFiles = false
 
 protobuf {
   protoc {
-    artifact = "com.google.protobuf:protoc:3.17.3"
+    artifact = "com.google.protobuf:protoc:3.19.1"
```

Comment:
So this introduces an incompatibility because we're using an older proto-java version. We should be declaring our proto java lib version (we get it transitively from grpc-protobuf today), but that will go viral quick so I'm not going to try to tackle that here.